mk/spksrc.cross-rust-env.mk

```diff
@@ -0,0 +1,62 @@
+# Configuration for rust build
+# 
+
+RUST_TOOLCHAIN ?= stable
+
+RUST_TARGET =
+# map archs to rust targets
+ifeq ($(findstring $(ARCH), $(x64_ARCHS)),$(ARCH))
+RUST_TARGET = x86_64-unknown-linux-gnu
+endif
+ifeq ($(findstring $(ARCH), $(i686_ARCHS)),$(ARCH))
+RUST_TARGET = i686-unknown-linux-gnu
+endif
+ifeq ($(findstring $(ARCH), $(ARMv5_ARCHS)),$(ARCH))
+# may be not supported for cargo
+RUST_TARGET = armv5te-unknown-linux-gnueabi
+endif
+ifeq ($(findstring $(ARCH), $(ARMv7_ARCHS)),$(ARCH))
+RUST_TARGET = armv7-unknown-linux-gnueabihf
+endif
```

Comment:
Is there a need for a ARMv7L as well?

Reply:
Afaik ARMv7_ARCHS includes arm7l, and armv7-unknown-linux-gnueabihf is the proper binary for armv7, armv7l, and armv8l